src/types/Time.ts

```diff
@@ -0,0 +1,29 @@
+import * as TypeUtils from '../TypeUtils'
+
+class Time implements SBVRType<string, Date> {
+	types = {
+		  postgres: 'TIME'
+		, mysql: 'TIME'
```

Comment:
Can you put the `,` at the end please, to avoid needing to mix spaces and tabs above